Sources/Preferences/Container.swift

```diff
@@ -27,14 +28,17 @@ extension Preferences {
 
 		- Parameters:
 			- contentWidth: A fixed width of the container's content (excluding paddings).
+			- minimumLabelWidth: A minimum width for labels within this container.
```

Comment:
Can you mention what the default behavior is? So people don't think they *have* to set this property to get the correct behavior.

Reply:
Ok, I've added information about the default value, and its behaviour 👍🏻